generatebundlefile/ecr.go

```diff
@@ -372,38 +348,41 @@ func (c *ecrClient) GetAuthToken() (string, error) {
 	return authToken, nil
 }
 
-// GetAuthConfig formats the Auth token to skopeo format, and adds to the ECR client struct
-func (c *ecrClient) GetAuthConfig() (*docker.AuthConfiguration, error) {
-	// Get ECR authorization token
-	authToken, err := c.GetAuthToken()
-	if err != nil {
-		return nil, errors.Cause(err)
-	}
+type DockerAuth struct {
+	Auths map[string]DockerAuthRegistry `json:"auths,omitempty"`
+}
 
-	// Decode authorization token to get credential pair
-	creds, err := base64.StdEncoding.DecodeString(authToken)
+type DockerAuthRegistry struct {
+	Auth string `json:"auth"`
+}
+
+func NewAuthFile(privToken, publicToken, accountID string) (string, error) {
```

Comment:
Probably good to add a comment here that says something to the effect of: "It's the caller's responsibility to dispose of the file that's created."